src/js/components/Select/__tests__/Select-test.js

```diff
@@ -61,8 +78,18 @@ describe('Select', () => {
       expect(document.activeElement).toMatchSnapshot();
       done();
     }, 100);
-  });
+    expect(onOpen).toHaveBeenCalled();
+    fireEvent.click(getByPlaceholderText('test select'));
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
```suggestion
    // Verifing the Drop is closed once the placholder eas clicked on
    expect(container.firstChild).toMatchSnapshot();
```

Reply:
Any reason we are not taking this commit?

Reply:
In the most recent push I added comments throughout the tests and there is a comment here